data/856/331/59/85633159.geojson

```diff
@@ -4,8 +4,16 @@
   "properties": {
     "edtf:cessation":"uuuu",
     "edtf:inception":"uuuu",
+    "eurostat:country_code":"UK",
```

Comment:
Confirming WOF maintains a source for this: https://github.com/whosonfirst/whosonfirst-sources/blob/main/sources/README.md#eurostat